system/Session/Handlers/DatabaseHandler.php

```diff
@@ -11,14 +11,17 @@
 
 namespace CodeIgniter\Session\Handlers;
 
+use CodeIgniter\Database\BaseBuilder;
 use CodeIgniter\Database\BaseConnection;
 use CodeIgniter\Session\Exceptions\SessionException;
 use Config\App as AppConfig;
 use Config\Database;
 use ReturnTypeWillChange;
 
 /**
- * Session handler using current Database for storage
+ * Base database session handler
+ *
+ * Do not use this class. Use database specific handler class.
```

Comment:
What are SQLite and MSSQL supposed to use?

Reply:
We now support only MySQL and PostgreSQL.

> Only MySQL and PostgreSQL databases are officially supported, due to lack of advisory locking mechanisms on other platforms. Using sessions without locks can cause all sorts of problems, especially with heavy usage of AJAX, and we will not support such cases. Use session_write_close() after you’ve done processing session data if you’re having performance issues.
https://codeigniter4.github.io/CodeIgniter4/libraries/sessions.html#databasehandler-driver

I think MSSQL would be supported if someone write a handler.